lib/mayaUsd/ufe/private/UfeNotifGuard.cpp

```diff
@@ -15,7 +15,6 @@
 //
 #include "UfeNotifGuard.h"
 
-#include <ufe/path.h>
```

Comment:
I noticed a few places where you have removed path.h - can you explain why? It is clearly used in this file - for example line 21. And path.h is **not** included by UfeNotifGuard.h. So it needed by this .cpp. It just so happens that it ends up being included down the line by <ufe/transform3d.h> but that is not something we should rely on. If a .cpp needs something it should include the header file.

Reply:
Yes I do realize that. I missed the class forwarding in UfeNotifGuard.h. The awkward formatting makes it hard to see it with a first glance. It is a few of them. 

I like the practice of going with forward declaration where it is appropriate.

Reply:
Yes I agree - forward declaration is the way to go. But then you still need to include <ufe.path.h> in this cpp.